langlib/lang.transaction/src/main/ballerina/transaction_errors.bal

```diff
@@ -19,6 +19,19 @@ public type Error distinct error;
 
 # Log and prepare `error` as a `Error`.
 #
+# ```ballerina
```

Comment:
We don't need this. This function being public is a bug: https://github.com/ballerina-platform/ballerina-lang/issues/38853.

Reply:
Thanks. Removed